src/extension/windows/windowsPlatform.ts

```diff
@@ -41,7 +41,8 @@ export class WindowsPlatform extends GeneralMobilePlatform {
             },
         };
 
-        extProps = TelemetryHelper.addReactNativeVersionToEventProperties(this.runOptions.reactNativeVersion, extProps);
+        extProps = TelemetryHelper.addReactNativeVersionToEventProperties(this.runOptions.reactNativeVersions.reactNativeVersion, extProps);
```

Comment:
It seems that we get the version twice here.

Reply:
I'll remove redundant getting of version.